### DISCUSSION TITLE
What's the reason brew can't add itself to PATH?

### DISCUSSION THREAD

**Discussion by DatGuyJonathan** [Homebrew/discussions#3681] [Getting started]
Brew spits out next steps at the end of installation. It says something like `Run these commands in your terminal to add Homebrew to your PATH`. If it knows which commands the user should run, why can't the install script actually do it?

**Comment by SMillerDev** [comment]
Because that would be very bad practice security wise. This way you can inspect the thing that'll get loaded into your she'll every time you start it, and decide if you want it.

**Reply by gromgit** [reply]
Also, the last instruction in that series sets up your active shell's environment to enable Homebrew. It's therefore pointless for the installer to run it, since it exits immediately after it prints those instructions.
